addons/binding/org.openhab.binding.feed.test/src/test/groovy/org/openhab/binding/feed/test/FeedHandlerTest.groovy

```diff
@@ -0,0 +1,629 @@
+        assertThat httpService,is(notNullValue())
+        servlet = new FeedServiceMock(DEFAULT_MOCK_CONTENT);
+        httpService.registerServlet(MOCK_SERVLET_PATH, servlet, null, null)
+    }
+
+    private void unregisterFeedTestServlet(){
+        HttpService httpService = getService(HttpService)
+        assertThat httpService,is(notNullValue())
+        httpService.unregister(MOCK_SERVLET_PATH)
+        servlet=null
+    }
+
+    @Before
+    public void setUp() {
+        volatileStorageService = new VolatileStorageService()
+        registerService(volatileStorageService)
+
+        managedThingProvider = getService(ThingProvider, ManagedThingProvider)
+        assertThat managedThingProvider, is(notNullValue())
+
+        thingRegistry = getService(ThingRegistry)
+        assertThat thingRegistry, is(notNullValue())
+
+        registerFeedTestServlet();
+        println ("Set Up finished");
+    }
+
+    private String generateURLString(String protocol,String hostname,int port,String path) {
+        def String mockServletURL = "$protocol://$hostname:$port$path";
+        return mockServletURL
+    }
+
+    private void initializeDefaultFeedHandler (){
+        def mockServletURL = generateURLString(MOCK_SERVLET_PROTOCOL,MOCK_SERVLET_HOSTNAME,MOCK_SERVLET_PORT,MOCK_SERVLET_PATH)
+        def defaultFormat = FeedBindingConstants.DEFAULT_FEED_FORMAT
+        //one minute update time is used for the tests
+        def defaultTestRefreshInterval = new BigDecimal(DEFAULT_TEST_AUTOREFRESH_TIME)
+        def defaultNumberOfEntries = new BigDecimal(FeedBindingConstants.DEFAULT_NUMBER_OF_ENTRIES)
+        initializeFeedHandler(mockServletURL,defaultTestRefreshInterval,defaultFormat,defaultNumberOfEntries)
+    }
+
+    private void initializeFeedHandler (String URL) {
+        initializeFeedHandler(URL,null,null,null)
+    }
+
+    private void initializeFeedHandler (String URL,BigDecimal numberOfEntries) {
+        initializeFeedHandler(URL,null,null,numberOfEntries)
+    }
+
+    private initializeFeedHandler(String URL,String format) {
+        initializeFeedHandler(URL,null,format,null)
+    }
+
+    private void initializeFeedHandler(String URL,BigDecimal refreshTime,String format,BigDecimal numberOfEntries) {
+        //set up configuration
+        Configuration configuration = new Configuration()
+        configuration.put((FeedBindingConstants.URL), URL)
+        configuration.put((FeedBindingConstants.REFRESH_TIME),refreshTime)
+        configuration.put((FeedBindingConstants.FEED_FORMAT), format)
+        configuration.put((FeedBindingConstants.NUMBER_OF_ENTRIES), numberOfEntries)
+
+        //create Feed Thing
+        ThingUID feedUID = new ThingUID(FeedBindingConstants.FEED_THING_TYPE_UID,THING_NAME)
+        channelUID = new ChannelUID(feedUID,FeedBindingConstants.FEED_CHANNEL)
+        feedThing = managedThingProvider.createThing(FeedBindingConstants.FEED_THING_TYPE_UID, feedUID, null, null, configuration)
```

Comment:
This method has change to "createThingofType" - please adapt your tests accordingly, thanks!